### PR TITLE
Lf 3023 deleted record splash screen

### PR DESCRIPTION
**Pull request by javiscript13**
**Description**

A splash screen was added to not found tasks and sensors  (the other objects got pending for futures releases).  

If a task or sensor is accessed by direct link and it was deleted or never existed it should take to the Unknown Record page.

The app at this time loads all tasks an sensors to the redux-store,  when you access one of them it loads the information on the store and does not make any api call, so for deleted tasks and sensors works only if you do it through the ui, if you delete the task on database and do not reload the app, it will show the data on the store (this was discussed on the ticket page).

Jira link: https://lite-farm.atlassian.net/browse/LF-3023

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

To test try to access a non existing sensor, or task; it should take you to the Unknown record page.

Please be aware that for the current data management of the app, if you delete on database and without reloading the app try to access the record, it will be shown as the data is retrieved from the redux-store.

- [x] Passes test case
- [x] UI components visually reviewed on desktop view
- [x] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [x] I have added or updated language tags for text that's part of the UI
- [x] I have added "MISSING" for all new language tags to languages I don't speak
- [x] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/src/containers/SensorReadings/index.jsx [hunk 1]
```diff
@@ -113,6 +122,7 @@ function SensorReadings({ history, match }) {
                 predictedXAxisLabel,
                 activeReadingTypes,
               } = sensorVisualizationPropList[type];
+              console.log('aqui', sensorVisualizationPropList[type]);
```
Comment (Duncan-Brain) [inline packages/webapp/src/containers/SensorReadings/index.jsx:125]: @javiscript13 just a console log here

File: packages/webapp/src/containers/LocationDetails/PointDetails/SensorDetail/index.jsx [hunk 2]
```diff
@@ -27,10 +27,14 @@ export default function SensorDetail({ history, user, match }) {
...
+
+  if (sensorInfo === undefined) {
+    history.replace('/unknown_record');
+  }
 
   useEffect(() => {
     dispatch(getSensorReadingTypes({ location_id }));
+    const partner_id = sensorInfo?.partner_id;
     dispatch(getSensorBrand({ location_id, partner_id }));
   }, []);
```
Comment (Duncan-Brain) [inline packages/webapp/src/containers/LocationDetails/PointDetails/SensorDetail/index.jsx:39]: @javiscript13 does this now need any dependencies?

**Comment by Duncan-Brain** [comment]
@javiscript13 

Looking good it seems to be working for me. 

1. A couple of inline questions
2. I am seeing this error on the tasks tab on sensorDetails:
Screenshot 2023-04-16 at 8 47 01 AM

3. For the splash screen it would be could if it were limited to 100% height of screen. I don't know how hard that would be so take it or leave it but because its not much content it could be good that you don't have to scroll to see the button.

File: packages/webapp/src/containers/SensorReadings/index.jsx (hunk 1 above)
Comment (Pazzolo) [reply packages/webapp/src/containers/SensorReadings/index.jsx]: fixed

**Comment by Pazzolo** [comment]
@Duncan-Brain Splash screen max height is fixed

**Comment by javiscript13** [comment]
@Duncan-Brain  all fixed.

**Review by Pazzolo — COMMENTED** [review]
LGTM

**Review by kikemarquez8 — DISMISSED** [review]

**Comment by Duncan-Brain** [comment]
@Pazzolo 

So at least on android it seems fine but on iPhone simulator on browser it still has scrolling. I don't think this is critical to release but would something like 100% not be better? or fit to container I don't get it.

@javiscript13 

Sensor seems to be working now! Am I understanding right that the selector isnt. filtering out deleted records for locations and sensors? We should fix that sometime. But tasks and sensors works for me!

**Review by Duncan-Brain — DISMISSED** [review]

**Review by kikemarquez8 — APPROVED** [review]

**Pull request merged**